scss/core/core.scss

```diff
@@ -1,4 +1,5 @@
 @import "css/variables";
+@import "css/custom-media";
```

Comment:
is the plan to add more than just breakpoints to this file? it seems we have a couple options going forward:
* add all media query related stuff to `custom-media` (media queries cover a [range of things](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries), not sure if following that is how we want to group our files or not)
* add all breakpoints (`max-width` queries) to `custom-breakpoints` (or a similarly named file with the same limited scope), add other media queries (`orientation` etc.) to other `.css` files

Reply:
It's great ideas! I think we have opportunities for flexible setup thanks to Post CSS and plugins "custom-media".

Reply:
I'd lean a bit more towards creating separate transforms for the different types of media queries, starting with the `custom-breakpoints` in this PR (though, maybe `custom-media-breakpoints`?).

In the future, we could expanding the set of custom media queries around things like device width, etc. (i.e., media queries that depend on CSS variable(s) generated from a design token).

Reply:
Yes, I think `custom-media-breakpoints` is a more obvious naming variant 👍